Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -324,7 +324,7 @@ await _dbContext.TestQuestion
                     question.CodeSnippetQuestion = Mapper.Map<CodeSnippetQuestionAC>(x.Question.CodeSnippetQuestion);
                     question.SingleMultipleAnswerQuestion = Mapper.Map<SingleMultipleAnswerQuestionAC>(x.Question.SingleMultipleAnswerQuestion);
                     //Removing correct answer(s)
-                    question.SingleMultipleAnswerQuestion.SingleMultipleAnswerQuestionOption.ForEach(y => y.IsAnswer = false);
+                    question.SingleMultipleAnswerQuestion?.SingleMultipleAnswerQuestionOption.ForEach(y => y.IsAnswer = false);
```

Comment:
we use ? operator only when assigning it to other variable or in if statement, this is not the proper way